azure-pipelines.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   variables:
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     DOTNET_CLI_TELEMETRY_OPTOUT: 1
-    PackageVersion: $[format('2.5.2-ci{0}', variables['Build.BuildId'])]
+    PackageVersion: $[format('2.5.3-ci{0}', variables['Build.BuildId'])]
```

Comment:
This diff doesn't make sense to me... This value should currently be 3.1.0....? Regardless, it shouldn't be changed in this PR